zerver/openapi/python_examples.py

```diff
@@ -826,12 +826,6 @@ def get_user_groups(client: Client) -> int:
     return leadership_user_group["id"]
 
 
-def test_user_not_authorized_error(nonadmin_client: Client) -> None:
-    result = nonadmin_client.get_streams(include_all=True)
-    assert_error_response(result)
-    validate_against_openapi_schema(result, "/rest-error-handling", "post", "400")
-
```

Comment:
I'm not sure what this is testing in the first place. Should this be removed since the current error is no longer applicable or should we create another scenario here that reproduces a `User not authorized for this query` error.

Reply:
Yeah, this seems not important. It was probably just the very first error handling test, or something like that.